lib/cli.rb

```diff
@@ -154,6 +154,14 @@ def self.handle_error(e)
     true
   end
 
+  def self.check_container_name!(image, name)
+    if image.include?("/") && name.include?("/")
+      raise Machinery::Errors::InvalidCommandLine.new(
+        "Error: Slashes are not allowed in machinery descriptions. Please provide a" \
+          " name for the description with the parameter `--name=NAME`."
```

Comment:
I'd state here what is the currently generated description name. We can reuse the message for invalid description names:

```
System description name 'mamorales/opensuse' is invalid. Only 'a-zA-Z0-9_:.-' are valid characters.
Please provide a valid name for the description with the parameter `--name=NAME`.
```